app/components/list-databases/component.js

```diff
@@ -0,0 +1,46 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  sortBy: ['handle:asc'],
+
+  arrayChanged: function() {
+    Ember.run.once(this, 'checkModelArray');
+  }.observes('model.[]'),
```

Comment:
The reason there's an `Ember.run.once` is that it will otherwise call `checkModelArray` for each element in the model, at least when I tested this.